src/JetReconstruction.jl

```diff
@@ -67,6 +68,14 @@ export tiled_jet_reconstruct
 include("EEAlgorithm.jl")
 export ee_genkt_algorithm
 
+## Tiling Base
+include("TilingBase.jl")
+export TilingBase
+
+## SoftKiller
+include("SoftKiller.jl")
+export SoftKiller, apply, select_ABS_RAP_max, push_data, plot_set_up
```

Comment:
```suggestion
export SoftKiller, apply, select_ABS_RAP_max
```
,`push_data` and `plot_set_up` are defined in examples and don't exist in the module

Reply:
I went with softkiller_apply but I can change it to just softkiller if you believe it could be more sufficient.